compiler/docgen.nim

```diff
@@ -211,9 +212,11 @@ proc newDocumentor*(filename: AbsoluteFile; cache: IdentCache; conf: ConfigRef,
       writeFile(outp, importStmt & content)
       let c = if cmd.startsWith("nim "): os.getAppFilename() & cmd.substr(3)
               else: cmd
-      let c2 = c % quoteShell(outp)
+      let subst = if "-r" in c: "-r" else: "$1"
+      let c2 = c.replace(subst, "--verbosity:0 --hints:off " & subst) % quoteShell(outp)
       rawMessage(conf, hintExecuting, c2)
       if execShellCmd(c2) != status:
+        echo importStmt & content
```

Comment:
No, the compiler doesn't use ``echo`` for error reporting and this PR is about making the compiler less noisy, not moreso.

Reply:
Idea was to print the snippet code that failed, makes it easier to debug the problematic snippet instead of having to search the snippet numerically in source file.

Removed for now.